src/cli/test.py

```diff
@@ -59,6 +62,23 @@ def print_text(line_number, text):
 else:
 	face_detector = dlib.get_frontal_face_detector()
 
+pose_predictor = dlib.shape_predictor(path + "/../dlib-data/shape_predictor_5_face_landmarks.dat")
+face_encoder = dlib.face_recognition_model_v1(path + "/../dlib-data/dlib_face_recognition_resnet_model_v1.dat")
+
+encodings = []
+models = None
+
+try:
+	user = builtins.howdy_user
+	models = json.load(open(path + "/../models/" + user + ".dat"))
+
+	for model in models:
+		encodings += model["data"]
+except FileNotFoundError:
+	print("No face model known for the user " + user + ", please run:")
+	print("\n\tsudo howdy -U " + user + " add\n")
+	#sys.exit(1)
```

Comment:
Did you want to remove this line initially? I think the current code is the best way of handling this error, notifying the user but not crashing

Reply:
yes, as currently the code do not request you a model, if model is not present it will only mark a face but not resolve any tag on model